server/app.js

```diff
@@ -19,11 +19,22 @@ app.use(urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(join(__dirname, "public")));
 
-
-
+const db = require("./helpers/db");
+const connection = db.getConnection();
+connection
+  .once('open', () => {
+    connection.db.listCollections().toArray((err, collections) => {
+      collections.forEach(item => {
+        if (item.name === 'questions') connection.db.dropCollection('questions');
+      });
+      questionService.seedQuestions()
+        .then((res) => {}); 
+    });
```

Comment:
Can you write a separate script to do seeding of the database? I just don't want this to affect the logic of the main app

Reply:
Okay